tedana/workflows/tedana.py

```diff
@@ -170,18 +190,30 @@ def _get_parser():
         help=(
             "Value used for random initialization of ICA "
             "algorithm. Set to an integer value for "
-            "reproducible ICA results. Set to -1 for "
-            "varying results across ICA calls. "
+            "reproducible ICA results (fastica/robustica). Set to -1 for "
+            "varying results across ICA (fastica/robustica) calls. "
```

Comment:
This is a bit too terse to be useful. If intent is something like "This applies to both fastica and robustica", say so more verbosely.